CONTRIBUTING.rst

```diff
@@ -31,11 +31,13 @@ Install Python requirements::
 
     pip install -r requirements/local.txt
 
-Copy the ``.env`` file somewhere that will be sourced when you need it::
+Copy the ``.env`` file to config/settings/.env::
 
-    cp env.example $VIRTUAL_ENV/bin/postactivate
+    cp env.example config/settings/.env
 
-Edit this file to fill in values for the missing settings.
+Edit this file to fill in values for the missing settings, especially
+GITHUB_USERNAME and GITHUB_PASSWORD. You will need to use a
+`Personal Access Token` instead of a password if you have 2-factor auth setup.
```

Comment:
Quibble: "...have 2-factor authentication set up." perhaps?

Reply:
And I think you need an underscore after the backticks for it to count as a reStructuredText link.

Reply:
Done and ready to review.